packages/stack/blockchain/src/index.js

```diff
@@ -30,17 +32,17 @@ export default class Blockchain {
     }
 
     this.blockchainNodes = {};
-    this.events.setCommandHandler("blockchain:node:register", (clientName, clientFunctions) => {
+    this.events.setCommandHandler("blockchain:node:register", (clientName, clientFunctions, cb = () => {}) => {
```

Comment:
If we don't pass a callback, we can't use `request2`? Sounds like a bug in `setCommandHandler`

Reply:
IIRC it just hangs without the callback.

Reply:
> If we don't pass a callback, we can't use request2? Sounds like a bug in `setCommandHandler`

It is definitely a shortcoming of the current implementation for `setCommandHandler`.